.circleci/config.yml

```diff
@@ -1,278 +1,62 @@
 version: 2
 jobs:
 
-  build-caches:
-    machine: true
+  lucene-and-others-javadoc:
+    docker:
+      - image: akmo03/groovy-jdk11:latest
+    parallelism: 1
```

Comment:
Why we stop using ability to run separate commands in parallel ?

it should stay 4, and please use same hack by commands.txt to keep each job on 4 paralel executions.
We have limit of parallel jobs, but not a containers.
![image](https://user-images.githubusercontent.com/812984/127785171-0cc5d847-e5c7-47e3-99e7-a93dd27be472.png)

Reply:
hmmm https://app.circleci.com/pipelines/github/AkMo3/checkstyle/133/workflows/7b7b5a00-ef69-4e7d-8219-784743a3c9bc
looks like if we do not use continaers, we still use paralleization
![image](https://user-images.githubusercontent.com/812984/127785844-9657da80-307b-4aea-b9a3-8e134f6df5bc.png)

Reply:
in this case usage of parallelzation in containers does not make benefit, lets keep config simple.

Reply:
Done